jscomp/main/jsoo_main.ml

```diff
@@ -180,7 +188,12 @@ let dir_directory d =
 let () =
   dir_directory "/static/cmis"
 
-let make_compiler name impl =
+module Converter = Refmt_api.Migrate_parsetree.Convert(Refmt_api.Migrate_parsetree.OCaml_404)(Refmt_api.Migrate_parsetree.OCaml_406)
+
```

Comment:
Is reason developed against 404? Maybe a good time to bump into 406 as well

Reply:
Should we open an issue on Reason to let them know that BS v7 and above won't have any active 4.02 builds anymore? Not sure what Reason planned to support on the OCaml side

Reply:
yeah, if they raise to 406 then such conversion is free?

Reply:
Apparently there seems to be a plan to upgrade. I created an issue to verify this:
[reason#2496](https://github.com/facebook/reason/issues/2496)